server/chat/backend/agent/prompt/composer.py

```
@@ -22,34 +22,45 @@
-    from chat.backend.agent.skills.loader import load_core_prompt
 
-    core_dir = os.path.join(
-        os.path.dirname(__file__), os.pardir, "skills", "core"
-    )
-    core_dir = os.path.normpath(core_dir)
+def _build_rca_system_prompt() -> str:
+    """Assemble the background RCA system prompt from rca_sections/*.md."""
+    parts = []
+    for name in _RCA_SECTION_ORDER:
+        path = os.path.join(_RCA_SECTIONS_DIR, f"{name}.md")
+        with open(path, "r", encoding="utf-8") as f:
```

Ruff (0.15.13)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)